### PR TITLE
Print error message if invalid sub command is given

### PR DESCRIPTION
**Pull request by michaelimfeld**
##### SUMMARY
Print error message if invalid sub command is given

##### ISSUE TYPE
 - Feature Pull Request

### CODE REVIEW

File: backupctl/backupctl.py [hunk 1]
```diff
@@ -142,6 +142,7 @@ def main():
     elif args.command == "log":
         history_show(hist)
     else:
+        LOG.error("backupctl: '{}' is not a backupctl command.".format(args.command))
```
Comment (fujexo) [inline backupctl/backupctl.py]: Can we print the help after this?

Comment (tongpu) [reply backupctl/backupctl.py]: I would rather add [choices](https://docs.python.org/2/library/argparse.html#choices) to the `add_argument` call for command.

**Comment by tongpu** [comment]
I believe this this is now broken, because of a wrong return code (2 instead of 1): https://github.com/adfinis-sygroup/backupctl/blob/master/backupctl/backupctl_test.py#L73
If you fix that the tests should become green again.

File: README.rst [hunk 2]
```diff
@@ -122,7 +122,7 @@ The following exit values are returned:
   An error occurred.
 
 2
-  No command was given.
+  No command or an invalid commad was given.
```
Comment (fujexo) [inline README.rst]: `s/commad/command/`

**Pull request merged**